### ISSUE TITLE
Warning Messages Babel/polyfill: Bump corejs to v3?

### ISSUE THREAD

**Issue by tacman** [adminkit/adminkit#106]
With Yarn 2, I"m getting a warning message about babel/polyfill.  (It's in AdminkitPro, but I think it has the same issue for the free version).

In my webpack, I'm using 

```js
    // enables @babel/preset-env polyfills
    .configureBabelPresetEnv((config) => {
        config.useBuiltIns = 'usage';
        config.corejs = 3;
    })
```

Here's the error:

```
    at C (/home/tac/ca/ac/.yarn/releases/yarn-berry.cjs:2:364565)
tac@tac-XPS-13-7390:~/ca/ac$ yarn install
➤ YN0000: ┌ Resolution step
➤ YN0061: │ @babel/polyfill@npm:7.12.1 is deprecated: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
➤ YN0061: │ core-js@npm:2.6.12 is deprecated: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.

```

In Yarn 1:

```
yarn install
yarn install v1.23.0-20210103.1434
[1/4] Resolving packages...
warning @symfony/webpack-encore > webpack-dev-server > sockjs > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning @symfony/webpack-encore > resolve-url-loader > rework > css > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning @symfony/webpack-encore > resolve-url-loader > rework > css > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning @symfony/webpack-encore > resolve-url-loader > rework > css > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning adminkit-pro > @babel/polyfill@7.12.1: 🚨 This package has been deprecated in favor of separate inclusion of a polyfill and regenerator-runtime (when needed). See the @babel/polyfill docs (https://babeljs.io/docs/en/babel-polyfill) for more information.
warning adminkit-pro > @babel/polyfill > core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
```

**Comment by paullaros** [comment]
Hey Tac, thanks! We'll address this issue in the next minor release.

**Comment by paullaros** [comment]
Resolved in #113
